client/src/main.ts

```diff
@@ -62,51 +123,84 @@ setupDiscordSdk().then(async () => {
 		view.msgList.pushMessage(msg)
 	})
 })
-	
-async function setupDiscordSdk() {
-	await discordSdk.ready()
-	console.log("Discord SDK is ready")
-		
-	// Authorize with Discord Client
-	const { code } = await discordSdk.commands.authorize({
-		client_id: import.meta.env.VITE_DISCORD_CLIENT_ID,
-		response_type: "code",
-		state: "",
-		prompt: "none",
-		scope: [
-			"identify",
-			"guilds",
-			"applications.commands"
-		]
+
+async function setupOIDC() {
+	api  = new ThreadletAPI("", {
```

Comment:
How is it supposed to auth without session key?
I know that this used to be a Discord-provided session key but the API *does* need *some* kind of key.

Reply:
I see you give it a session key below, but this instantiates two instances of the API client. Instead, delete this and make two consts and use `GATEWAY` instead of `api.GATEWAY` in line 160.

```suggestion
	const API_ROOT = "/api/v0"
	const GATEWAY = "/api/v0/gateway"
```

Reply:
Fixed by [8e6739](https://github.com/j0code/threadlet/pull/1/commits/8e674394212796d88d08ed31d6dd0915ec533db8)